src/Core/Domain/Tax/Command/BulkToggleTaxStatusCommand.php

```diff
@@ -89,7 +89,7 @@ private function setTaxIds(array $taxIds)
     /**
      * Validates that value is of type boolean
      *
-     * @param $value
+     * @param bool $value
```

Comment:
Same logic here. This method `assertIsBool` very purpose is to check that given variable is a boolean. It makes no sense to validate it if we assume it's a boolean 😉

Reply:
I agree. I put a `mixed` type.

Reply:
:heavy_check_mark: